ovos_utils/log.py

```diff
@@ -145,9 +152,9 @@ def init_service_logger(service_name):
     # this is makes all logs from this service be configured to write to service_name.log file
     # if this is not called in every __main__.py entrypoint logs will be written
     # to a generic OVOS.log file shared across all services
-    from ovos_config import Configuration
+    from ovos_config.config import read_mycroft_config
```

Comment:
Should this be accepted as an arg? Should we fallback to some default config?

Reply:
#130 makes this reference safe, but if `ovos-config` is missing then defaults will always be used.